include/arch/arm/cortex_m/mpu/arm_mpu.h

```diff
@@ -155,6 +155,8 @@ struct arm_mpu {
 #define REGION_1G       (0x1D << 1)
 #define REGION_2G       (0x1E << 1)
 #define REGION_4G       (0x1F << 1)
+#define REGION_24K      (0x20 << 1)
```

Comment:
@galak @agross-linaro @ioannisg please review this change

Reply:
This is not correct. These macros are used to encode the region size in the MPU register, when configuring MPU regions, and AFAIK, we only have powers of two, there.
Check table B3-43 in ARMv7-M technical reference manual.